tools/cldr-apps/js/src/esm/cldrCoverage.js

```diff
@@ -48,14 +48,14 @@ function covName(lev) {
 }
 
 function effectiveCoverage(locale) {
-  const orgCov = getSurveyOrgCov(locale);
-  if (!orgCov) {
-    throw new Error(`surveyOrgCov(${locale}) not yet initialized`);
-  }
   const userCov = getSurveyUserCov();
   if (userCov) {
     return covValue(userCov);
   } else {
+    const orgCov = getSurveyOrgCov(locale);
+    if (!orgCov) {
+      throw new Error(`surveyOrgCov(${locale}) not yet initialized`);
+    }
```

Comment:
Supposing this `throw` happens in spite of not being supposed to happen, do we `catch` it anywhere? If not, I wonder if Survey Tool would recover more gracefully with or without the throw? I'd be tempted to print an error to the console then return a fallback value -- just a thought

Reply:
yes, it probably needs to be caught… we don't have a catch tree on the vue side.